lib/node_modules/@stdlib/repl/lib/complete_fs.js

```diff
@@ -128,6 +133,48 @@ function complete( out, expression, alias, path ) {
 			continue;
 		}
 	}
+	out.sort();
+	if ( !isFuzzy ) {
+		return filter;
+	}
+	// Start searching for fuzzy completions...
```

Comment:
This could potentially lead to an enormous number of potential completions. File system access isn't cheap, so I am not sure the best approach. It could be that we'd require matches to meet a higher threshold.

Reply:
If this carries performance risks, we can stick to exact matches in this case?

Reply:
May be worth seeing how other command-line tools supporting fuzzy walking of the file system handle this. I am sure there have been more than one tool written in Rust for this purpose. Maybe they restrict the number of matches more stringently?